wdio.login.conf.js

```diff
@@ -0,0 +1,8 @@
+import { config as baseConfig } from './wdio.conf.js';
```

Comment:
@karenn32 Is this one required at all? Here, those options for the command line has wdio:
--suite               overwrites the specs attribute and runs the defined
                        suite                                            [array]
--spec                run only a certain spec file - overrides specs piped
                        from stdin